CHANGES.txt

```diff
@@ -5,7 +5,7 @@ CHANGES
 4.1.0 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- BUGFIX: Use `base64.b64encode` to avoid deprecation warning with Python 3.
```

Comment:
I don't know if this can be callled 'BUGFIX', when it's just avoiding a (hidden by default) deprecation warning.